static/src/javascripts/projects/common/modules/navigation/edition-picker.js

```diff
@@ -0,0 +1,67 @@
+define([
+    'common/utils/fastdom-promise',
+    'common/utils/$'
+], function (
+    fastdomPromise,
+    $
+) {
+    var dropdown = $('.js-edition-picker-dropdown');
+
+    function enhanceToButton() {
```

Comment:
Well, this is just beautiful, my only bother when reading the mark-up that it wasn't a button, but lo-and-behold you have it covered ✊